VideoWeb/VideoWeb.UnitTests/Controllers/TokenController/GetTokenTests.cs

```diff
@@ -17,7 +17,7 @@ public void should_return_ok_token_response()
             typedResult.Should().NotBeNull();
             var tokenResponse = (TokenResponse)typedResult.Value;
             tokenResponse.Token.Should().Be(token);
-            tokenResponse.ExpiresOn.Length.Should().Be(17);
+            tokenResponse.ExpiresOn.Length.Should().BeGreaterThan(15);
```

Comment:
Things is that the length of token could be 16,  I had the error in the test on the pipline build and debug it find out that ExpiresOn token is 16 because of time  9 not 09, it's ok after 12am will be 17

Reply:
good eye